attach/CMakeLists.txt

```diff
@@ -5,5 +5,7 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
   endif()
 add_subdirectory(base_attach_impl)
 add_subdirectory(frida_uprobe_attach_impl)
-add_subdirectory(syscall_trace_attach_impl)
-add_subdirectory(text_segment_transformer)
+if(${BPFTIME_BUILD_WITH_LIBBPF})
```

Comment:
Use a name like "BPFTIME_BUILD_ON_X86" or somethings alike to control these two folders would be better

Reply:
Sure will do.

Reply:
in my recent commits, I am still trying to compile it, I will rename it once it is working fine.

Reply:
should I create another option for this or just define a new spec for when it is an apple system and exclude these files?

Reply:
I have used option like `if(UNIX NOT APPLE)`
let me know if you have think otherwise.